src/vmm/src/vmm_config/entropy.rs

```diff
@@ -90,7 +107,7 @@ mod tests {
     #[test]
     fn test_set_device() {
         let mut builder = EntropyDeviceBuilder::new();
-        let device = Entropy::new().unwrap();
+        let device = Entropy::new(RateLimiter::default()).unwrap();
```

Comment:
Nit: Could `Entropy` implement `Default`?
```suggestion
        let device = Entropy::default().unwrap();
```